### PR TITLE
fix line 111: Avoid reassigning parameters such as 'style'

### PR DESCRIPTION
**Pull request by tsvetaevAlex**

### CODE REVIEW

File: jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/Slider.java [hunk 1]
```diff
@@ -109,10 +109,12 @@ public static int getInteger(String attr, WebElement el, int defaultValue) {
...
-    style = style.replaceAll("[-?0-9]+", "");
-    int perInx = style.indexOf("%");
-    return style.substring(0, perInx) + 0 + style.substring(perInx, style.length()-2)
-        + value + style.substring(style.length()-2);
+    String styleLocal;
+    styleLocal= style.replaceAll("[-?0-9]+", "");
+    int perInx;
+    perInx = styleLocal.indexOf("%");
+    return styleLocal.substring(0, perInx) + 0 + style.substring(perInx, style.length() - 2)
+            + value + style.substring(style.length() - 2);
```
Comment (evasileva-epam) [inline jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/Slider.java:117]: It seems that style variable should be replaced by styleLocal on lines 116 and 117

File: jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/Slider.java [hunk 2]
```diff
@@ -109,10 +109,12 @@ public static int getInteger(String attr, WebElement el, int defaultValue) {
...
 
   private String setNewStyle(String style, int value) {
-    style = style.replaceAll("[-?0-9]+", "");
-    int perInx = style.indexOf("%");
-    return style.substring(0, perInx) + 0 + style.substring(perInx, style.length()-2)
-        + value + style.substring(style.length()-2);
+    String styleLocal;
+    styleLocal= style.replaceAll("[-?0-9]+", "");
+    int perInx;
+    perInx = styleLocal.indexOf("%");
```
Comment (evasileva-epam) [inline jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/Slider.java:115]: For what we need to update red line 113 to 2 new green lines 114 and 115?

**Review by evasileva-epam — CHANGES_REQUESTED** [review]

File: jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/Slider.java (hunk 1 above)
Comment (tsvetaevAlex) [reply jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/Slider.java]: updated

File: jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/Slider.java (hunk 2 above)
Comment (tsvetaevAlex) [reply jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/Slider.java]: concatenated into one line

**Pull request closed without merging**